entity.go

```diff
@@ -182,6 +206,20 @@ func (id *IndexDefinition) Clone() *IndexDefinition {
 	}
 }
 
+func (id *IndexDefinition) String() string {
+	return fmt.Sprintf("%+v", *id)
```

Comment:
This seems a bit shaky to me, if the `fmt` package makes a breaking change we could be in trouble.

Reply:
Why would `fmt` make a breaking change? We would be in trouble if _any_ of the standard packages made a breaking change....

Reply:
I agree that it's an unlikely concern but something to consider. From the pkg docs, it appears to format `%v` in the following way:

```
struct:             {field0 field1 ...}
array, slice:       [elem0 elem1 ...]
maps:               map[key1:value1 key2:value2 ...]
pointer to above:   &{}, &[], &map[]
```

For `ColumnDefinitions`, this will include the `Tags` field - which may have a random ordering. If we compare two equivalent `ColumnDefinitions#String` outputs we may have different results right. I don't full understand how the String and Equals methods are consumed so I'll defer to your judgement.

Reply:
That is an excellent point.  I don't know if it makes any actual difference but I'm not a fan of non-deterministic code so I'm going to fix that.

Reply:
Changed `ColumnDefinition.String()` to be deterministic.